File: UVA/10215_The_Largest_Smallest_Box/solution.py
import math

while 1:
    try:
        l, w = map(int, input().split())
    except:
        break

    tmp = math.pow(l, 2) + math.pow(w, 2) - l * w
    x = ((l + w) - math.sqrt(tmp)) / 6
    min_1 = min(l, w) / 2

    print(f"{x:.3f} 0.000 {min_1:.3f}")

File: UVA/10215_The_Largest_Smallest_Box/solution.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    double l, w;
    while (cin >> l >> w) {
        double tmp = pow(l, 2) + pow(w, 2) - l * w;
        double x = ((l + w) - sqrt(tmp)) / 6;
        double min_1 = min(l, w) / 2;

        cout << fixed << setprecision(3);
        cout << x << " 0.000 " << min_1 << endl;
    }

    return 0;
}